apps/server/src/modules/tool/school-external-tool/controller/dto/school-external-tool-configuration.response.ts

```diff
@@ -8,7 +8,14 @@ export class SchoolExternalToolConfigurationStatusResponse {
 	})
 	isOutdatedOnScopeSchool: boolean;
 
+	@ApiProperty({
+		type: Boolean,
+		description: 'Is the tool deactivated, because of school administrator?',
```

Comment:
assuming you read this description in swagger, can you do something with it? Maybe we should write that the school admin can deactivate a tool and then it can no longer be started